stream-chat-android/src/main/java/com/getstream/sdk/chat/view/channels/ChannelsView.kt

```diff
@@ -36,17 +35,40 @@ public class ChannelsView @JvmOverloads constructor(
             isVisible = false
             addView(loadingView, defaultChildLayoutParams())
         }
+        parseAttrs(attrs)
+    }
+
+    private fun parseAttrs(attrs: AttributeSet?) {
+        context.obtainStyledAttributes(attrs, R.styleable.ChannelsView, 0, 0).use {
+            it.getText(R.styleable.ChannelsView_streamChannelsEmptyStateLabelText)?.let { emptyStateText ->
+                emptyStateView.apply {
+                    if (this is TextView) {
+                        text = emptyStateText
+                    }
+                }
+            }
+        }
     }
 
+    /**
+     * @param view will be added to the view hierarchy of [ChannelsView] and managed by it.
+     * The view should not be added to another [ViewGroup] instance elsewhere.
+     * @param layoutParams defines how the view will be situated inside it's container ViewGroup.
```

Comment:
```suggestion
     * @param layoutParams defines how the view will be situated inside its container ViewGroup.
```

sorry 😅 

Good docs though, thank you for adding them!

Reply:
My bad 👍🏼